spec/requests/waste_exemptions_engine/exemptions_forms_spec.rb

```diff
@@ -4,7 +4,10 @@
 
 module WasteExemptionsEngine
   RSpec.describe "Exemptions Forms", type: :request do
-    before(:context) { create_list(:exemption, 5) }
+    before(:context) do
+      WasteExemptionsEngine::Exemption.delete_all
```

Comment:
What does this change do? Is this so we can be sure we are getting a certain number of exemptions?

Reply:
Yes. Especially for Exemptions, the HTML is generated based on the content of the database. I have noticed that those tests populating the DB with a before context were flakey, and the reason was in the number of exemptions that wasn't always 5. 
I had to put a dot somewhere so I haven't spent too long on finding out which test is leaving the DB dirty and why the DB cleaner is not picking it up. One for the next time :)